radsat-sk/operation/subsystems/RAntenna.h

```diff
@@ -7,18 +7,28 @@
 #ifndef RANTENNA_H_
 #define RANTENNA_H_
 
+#include <stdint.h>
+
 /***************************************************************************************************
                                             DEFINITIONS
 ***************************************************************************************************/
 
-/** I2C Slave Address for Transceiver Receive Port */
-#define ANTENNA_I2C_SLAVE_ADDR (0x31)
+/** I2C Slave Address for Antenna Receive Port */
+#define ANTENNA_I2C_SLAVE_ADDR_PRIMARY (0x31)
+
+typedef struct _antennaDeploymentStatus {
+	int DeployedsideA;
+	int DeployedsideB;
+} antennaDeploymentStatus;
 
 
 /***************************************************************************************************
                                              PUBLIC API
 ***************************************************************************************************/
 
+int antennaInit(void);
+int antennaDeployment(void);
+int antennaGetTemperature(uint16_t side);

```

Comment:
Do we ever care about _just_ temperature? Or should we just provide a full `antenna_telemetry_t` struct via an `antennaTelemetry()` function? Maybe we do care about just temperature, but regardless there should also be a telemetry struct (that's public) and a function that fills a pointer to one, since that's the direction we've been going with the other subsystems